Tools/CMake/AMReXBuildInfo.cmake

```diff
@@ -67,8 +67,8 @@ function (generate_buildinfo _target _git_dir)
    # Set variables to be used in generated source file
    #
 
-   # Date and time
-   string(TIMESTAMP BUILD_DATE "%Y-%m-%d %H:%M:%S"  )
+   # Date and time -- C++ preprocessor macros
+   set(BUILD_DATE "__DATE__ \" \"  __TIME__")
```

Comment:
Can we make this a cache-eable variable? At least we do not need to actively counter efforts for reproducible builds (Debian et al.) :-)

Reply:
Just added some codee to take care of that